.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,35 @@
+name: Bug Report
```

Comment:
This is still more or less a duplicate of qsl bug report. I would really make sure this focuses on something being wrong with the game, *not* mentioning bug